messaging-modules/spring-apache-camel/pom.xml

```diff
@@ -77,7 +77,7 @@
     <properties>
         <camel.version>4.3.0</camel.version>
         <camel-swagger.version>3.22.0</camel-swagger.version>
-        <junit-jupiter.version>5.10.1</junit-jupiter.version> <!--Version not compatible with the one inherited from parent-->
+        <junit-jupiter.version>5.10.2</junit-jupiter.version> <!--Version not compatible with the one inherited from parent-->

```

Comment:
The parent module does not seem to have any junit-jupiter property set, can you remove the comment here please. Thanks!